<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>new_file_dialog</class>
 <widget class="QDialog" name="new_file_dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>320</width>
    <height>240</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>new file</string>
  </property>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>61</x>
     <y>52</y>
     <width>207</width>
     <height>115</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>password:</string>
            </property>
            <property name="buddy">
             <cstring>userNameInput</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userNameInput"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>username:</string>
            </property>
            <property name="buddy">
             <cstring>userPasswordInput</cstring>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="userPasswordInput"/>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_4">
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>备注</string>
          </property>
          <property name="buddy">
           <cstring>userMemoryInput</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="userMemoryInput"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="NewFileOKButton">
        <property name="text">
         <string>OK</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="NewFileCancelButton">
        <property name="text">
         <string>Cancel</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <zorder>layoutWidget</zorder>
  <zorder>layoutWidget</zorder>
  <zorder>label_3</zorder>
  <zorder>userMemoryInput</zorder>
  <zorder></zorder>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>NewFileCancelButton</sender>
   <signal>clicked()</signal>
   <receiver>new_file_dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>266</x>
     <y>222</y>
    </hint>
    <hint type="destinationlabel">
     <x>283</x>
     <y>168</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
